exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlWorker.java

```diff
@@ -61,7 +61,8 @@ public static PhysicalPlan getPlan(QueryContext context, String sql, Pointer<Str
         context.getNewDefaultSchema(),
         context.getDrillOperatorTable(),
         (UdfUtilities) context,
-        context.getFunctionRegistry());
+        context.getFunctionRegistry(),
+        context.getOptions());
```

Comment:
SqlConverter already has pass in PlannerSettings(), why would you pass in another option manager?